fastcrypto-vdf/src/vdf/wesolowski/fiat_shamir.rs

```diff
@@ -1,46 +1,59 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::math::hash_prime::{hash_prime, PrimalityCheck};
+use crate::class_group::discriminant::Discriminant;
+use crate::class_group::QuadraticForm;
+use crate::math::hash_prime::hash_prime;
+use crate::math::parameterized_group::ParameterizedGroupElement;
 use crate::vdf::wesolowski::WesolowskisVDF;
-use crate::{ParameterizedGroupElement, ToBytes, UnknownOrderGroupElement};
+use fastcrypto::groups::multiplier::ScalarMultiplier;
 use num_bigint::BigInt;
-use std::marker::PhantomData;
+use serde::Serialize;
 
-pub trait FiatShamir<G: ParameterizedGroupElement + UnknownOrderGroupElement>: Sized {
+/// Default size in bytes of the Fiat-Shamir challenge used in proving and verification.
+pub const DEFAULT_CHALLENGE_SIZE_IN_BYTES: usize = 32;
```

Comment:
i don't remember - does it affect performance in any way? in theory it can be smaller

Reply:
if we stay with this length, let's document the security parameter we get and why

Reply:
I've added a reference to the paper and an explanation.